appveyor.yml

```diff
@@ -120,6 +120,15 @@ test_script:
      --timeout 60
      --n-retries 2
      test\command\suite
+  - ruby test\command\bin\grntest
+     --groonga %FULL_GROONGA_INSTALL_FOLDER%\bin\groonga.exe
+     --base-directory test\command
+     --reporter mark
+     --n-workers 1
```

Comment:
In my opinion, `--n-workers` are multiple better.
Because we are easy to find problems.

Reply:
It is reasonable to change.